src/lib/Libwin/gethostid.c

```diff
@@ -147,8 +147,10 @@ gethostid(void)
 
 	sz = sizeof(cname);
 
-	if (GetComputerName(cname, &sz) == 0)
+	if (GetComputerName(cname, &sz) == 0) {
+		log_err(-1, __func__, "failed in GetComputerName");
 		return (ret);
+	}
 
 	sz1 = 0;
 	sz2 = sizeof(domain);
```

Comment:
No log messages for other call failures - LookupAccountName, malloc,

Reply:
For malloc one, yes i will add log. But for others its already handled in the code

Reply:
Im not able to find the place where error logging of two invocations of LookupAccountName() is happening?

Reply:
also should we log error for any failures of 3 invocations of Netbios() ?

Reply:
actually LookupAccountName error handling path is covered in existing code. And for Netbios if that fails then we assume the id as pbs_server_addr. So don't think we need any extra login there. Let me know your thoughts